### PR TITLE
"CAMEL-19341:camel-spring-boot - Example with ActiveMQ Artemis"

### PR DESCRIPTION
**Pull request by ramu11**
camel-spring-boot - Example with ActiveMQ Artemis

### CODE REVIEW

**Comment by ramu11** [comment]
@davsclaus  Request to review

File: artemis/src/test/java/sample/camel/SampleAmqApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@CamelSpringBootTest
+@SpringBootTest(classes = SampleAmqApplication.class)
+public class SampleAmqApplicationTests {
+    @Autowired
+    private CamelContext camelContext;
+
+    @Disabled("Requires a running activemq broker")
```
Comment (essobedo) [inline artemis/src/test/java/sample/camel/SampleAmqApplicationTests.java:37]: Not sure if it makes sense to provide one single test that is disabled by default, either remove the test or launch artemis in embedded mode or using testcontainers with quay.io/artemiscloud/activemq-artemis-broker as image

**Review by essobedo — COMMENTED** [review]
Thx for the contribution, could you please remove the double quotes from your commit message?

File: artemis/src/test/java/sample/camel/SampleAmqApplicationTests.java (hunk 1 above)
Comment (ramu11) [reply artemis/src/test/java/sample/camel/SampleAmqApplicationTests.java:37]: ok I will Remove the test.

**Pull request closed without merging**